react/DynamicIframe.tsx

```diff
@@ -11,12 +11,22 @@ const DynamicIframe: StorefrontFunctionComponent<DynamicIframeProps> = ({
   const {
     route: { params },
   } = useRuntime()
-
+  var allParamsExist = true
   const src = dynamicSrc.replace(/({[A-z0-9]*})/g, function(match: string) {
     const thisParam = match.replace(/{|}/g, '')
-    return params[thisParam]
+    if (thisParam && params[thisParam]) {
```

Comment:
I would flip this condition. Keeping the happy path as the default path and the edge cases inside an if statement

```jsx
if (!thisParam || !params[thisParam]) {
  allParamsExist = false
  console.error(
    'parameter ' + thisParam + ' not found in runtime params: ' + params
  )
  return ''
}

return params[thisParam]
```

Reply:
More on this:
https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88